include/fixed_containers/int_math.hpp

```diff
@@ -94,4 +94,10 @@ constexpr UnsignedIntegralAndSign<T> safe_add(const T addend_left,
     return UnsignedIntegralAndSign<T>::create_positive(addend_left + static_cast<T>(addend_right));
 }
 
+template <std::integral T>
+constexpr T divide_integers_rounding_up(const T dividend, const T divisor)
+{
+    return ((dividend - static_cast<T>(1)) / divisor) + static_cast<T>(1);
```

Comment:
```
(div + divi - 1)/divi
```

Reply:
Possibly should assert for `0`?

Reply:
> (div + divi - 1)/divi
> (div - 1) / divi + 1

I used the second form because it is more resilient to overflow.


> Possibly should assert for 0?

Done.